styles.css

```diff
@@ -156,66 +162,85 @@ header {
         background: var(--white);
         width: 225px;
         height: 100vh;
-      }
+    }
 
-      .grid-layout {
+    .grid-layout {
         grid-template-areas:
-        "nav nav nav nav nav nav"
-        "main main main main main main"
-        "main main main main main main"
-        "main main main main main main"
-        "main main main main main main";
-      }
+            "nav nav nav nav nav nav"
+            "main main main main main main"
+            "main main main main main main"
+            "main main main main main main"
+            "main main main main main main";
+    }
 
-      .main {
-          padding: 1.5rem 4rem;
-      }
+    .main {
+        padding: 1.5rem 4rem;
+    }
 
-      .my-hamburger {
-          display: block;
-      }
+    .my-hamburger {
+        display: block;
+    }
 
-      .close-sidebar {
+    .close-sidebar {
         display: block;
-      }
+    }
 
-  }
+}
 
-  @media screen and (max-width: 640px) {
+@media screen and (max-width: 640px) {
     .main {
         padding: 1.5rem 1rem;
     }
 
-    .nav-section > .anchor {
+    .nav-section>.anchor {
         display: none;
     }
 }
+
+.component-example-container>.list-group {
+    max-width: 30rem;
+}
+
 /* --------------------- Modification for highlightjs ---------------- */
 
 pre {
     display: flex;
     margin-bottom: 3rem;
 }
 
-pre code.hljs{
-    -ms-overflow-style: none; /* Edge, Internet Explorer */
-    scrollbar-width: none; /* Firefox */
+pre code.hljs {
+    -ms-overflow-style: none;
+    /* Edge, Internet Explorer */
+    scrollbar-width: none;
+    /* Firefox */
     border: 2px solid var(--light-gray);
     width: 100%;
 }
-  
+
 pre code.hljs::-webkit-scrollbar {
-    display: none; /* Chrome, Safari, Opera */
+    display: none;
+    /* Chrome, Safari, Opera */
 }
 
-.hljs{
+.hljs {
     background: #f8f8f8 !important;
```

Comment:
I was told that we should avoid using Important and look where is it taking its color from and then change it